package/dropbear/0006-linkedlist.h-added.patch.conditional

```diff
@@ -0,0 +1,172 @@
+--- a/linkedlist.h	2019-07-11 14:07:04.040000000 +0530
```

Comment:
Patch needs appropriate format-patch header, including commit log providing a reason for the patch. For more info see: https://git-scm.com/docs/git-format-patch

Reply:
Yes, made the corrections and now created the patch with git format-patch header.